oauth2/cookies_test.go

```diff
@@ -133,7 +138,8 @@ func TestCookieValidate(t *testing.T) {
 				ValidErr: test.ValidErr,
 			},
 		}
-		principal, err := cook.Validate(context.Background(), req)
+		w := httptest.NewRecorder()
```

Comment:
Did you want to make assertions on `w` (that make use of `NewRecorder`)? @timraymond says "Maybe this is just a convenient way to create a noop response writer."